www-client/chromium/chromium-75.0.3770.80.ebuild

```diff
@@ -0,0 +1,718 @@
+			pax-mark m "out/Release/host/${x}"
+		else
+			eninja -C out/Release "${x}"
+			pax-mark m "out/Release/${x}"
+		fi
+	done
+
+	# Even though ninja autodetects number of CPUs, we respect
+	# user's options, for debugging with -j 1 or any other reason.
+	eninja -C out/Release chrome chromedriver
+	use suid && eninja -C out/Release chrome_sandbox
+
+	pax-mark m out/Release/chrome
+}
+
+src_install() {
+	local CHROMIUM_HOME="/usr/$(get_libdir)/chromium-browser"
+	exeinto "${CHROMIUM_HOME}"
+	doexe out/Release/chrome
+
+	if use suid; then
+		newexe out/Release/chrome_sandbox chrome-sandbox
+		fperms 4755 "${CHROMIUM_HOME}/chrome-sandbox"
+	fi
+
+	doexe out/Release/chromedriver
+
+	local sedargs=( -e "s:/usr/lib/:/usr/$(get_libdir)/:g" )
+	sed "${sedargs[@]}" "${FILESDIR}/chromium-launcher-r3.sh" > chromium-launcher.sh || die
+	doexe chromium-launcher.sh
+
+	# It is important that we name the target "chromium-browser",
+	# xdg-utils expect it; bug #355517.
+	dosym "${CHROMIUM_HOME}/chromium-launcher.sh" /usr/bin/chromium-browser
+	# keep the old symlink around for consistency
+	dosym "${CHROMIUM_HOME}/chromium-launcher.sh" /usr/bin/chromium
+
+	dosym "${CHROMIUM_HOME}/chromedriver" /usr/bin/chromedriver
+
+	# Allow users to override command-line options, bug #357629.
+	insinto /etc/chromium
+	newins "${FILESDIR}/chromium.default" "default"
+
+	pushd out/Release/locales > /dev/null || die
+	chromium_remove_language_paks
+	popd
+
+	insinto "${CHROMIUM_HOME}"
+	doins out/Release/*.bin
+	doins out/Release/*.pak
+	doins out/Release/*.so
+
+	if ! use system-icu; then
+		doins out/Release/icudtl.dat
+	fi
+
+	doins -r out/Release/locales
+	doins -r out/Release/resources
+
+	if [[ -d out/Release/swiftshader ]]; then
+		insinto "${CHROMIUM_HOME}/swiftshader"
+		doins out/Release/swiftshader/*.so
+	fi
+
+	# Install icons and desktop entry.
+	local branding size
+	for size in 16 22 24 32 48 64 128 256 ; do
+		case ${size} in
+			16|32) branding="chrome/app/theme/default_100_percent/chromium" ;;
+				*) branding="chrome/app/theme/chromium" ;;
+		esac
+		newicon -s ${size} "${branding}/product_logo_${size}.png" \
+			chromium-browser.png
+	done
+
+	local mime_types="text/html;text/xml;application/xhtml+xml;"
+	mime_types+="x-scheme-handler/http;x-scheme-handler/https;" # bug #360797
+	mime_types+="x-scheme-handler/ftp;" # bug #412185
+	mime_types+="x-scheme-handler/mailto;x-scheme-handler/webcal;" # bug #416393
+	make_desktop_entry \
+		chromium-browser \
+		"Chromium" \
+		chromium-browser \
+		"Network;WebBrowser" \
+		"MimeType=${mime_types}\nStartupWMClass=chromium-browser"
+	sed -e "/^Exec/s/$/ %U/" -i "${ED}"/usr/share/applications/*.desktop || die
+
+	# Install GNOME default application entry (bug #303100).
+	insinto /usr/share/gnome-control-center/default-apps
+	newins "${FILESDIR}"/chromium-browser.xml chromium-browser.xml
+
+	readme.gentoo_create_doc
+}
+
+pkg_postrm() {
+	if type gtk-update-icon-cache &>/dev/null; then
```

Comment:
It seems that usage of `xdg_icon_cache_update` was accidentally reverted.

Between chromium-73.0.3683.86.ebuild and chromium-74.0.3729.169.ebuild:
```
--- chromium-73.0.3683.86.ebuild
+++ chromium-74.0.3729.169.ebuild
@@ -699,20 +702,12 @@
 }
 
 pkg_postrm() {
-       if type gtk-update-icon-cache &>/dev/null; then
-               ebegin "Updating GTK icon cache"
-               gtk-update-icon-cache "${EROOT}/usr/share/icons/hicolor"
-               eend $?
-       fi
+       xdg_icon_cache_update
        xdg_desktop_database_update
 }
 
 pkg_postinst() {
-       if type gtk-update-icon-cache &>/dev/null; then
-               ebegin "Updating GTK icon cache"
-               gtk-update-icon-cache "${EROOT}/usr/share/icons/hicolor"
-               eend $?
-       fi
+       xdg_icon_cache_update
        xdg_desktop_database_update
        readme.gentoo_print_elog
 }
```
Between chromium-74.0.3729.169.ebuild and chromium-75.0.3770.80.ebuild:
```
--- chromium-74.0.3729.169.ebuild
+++ chromium-75.0.3770.80.ebuild
@@ -702,12 +699,20 @@
 }
 
 pkg_postrm() {
-       xdg_icon_cache_update
+       if type gtk-update-icon-cache &>/dev/null; then
+               ebegin "Updating GTK icon cache"
+               gtk-update-icon-cache "${EROOT}/usr/share/icons/hicolor"
+               eend $?
+       fi
        xdg_desktop_database_update
 }
 
 pkg_postinst() {
-       xdg_icon_cache_update
+       if type gtk-update-icon-cache &>/dev/null; then
+               ebegin "Updating GTK icon cache"
+               gtk-update-icon-cache "${EROOT}/usr/share/icons/hicolor"
+               eend $?
+       fi
        xdg_desktop_database_update
        readme.gentoo_print_elog
 }
```